app/models/office.rb

```diff
@@ -52,11 +54,28 @@ def as_json(options = nil)
   end
   # rubocop:enable Metrics/AbcSize
 
+  def as_relation_json
+    {
+      id:,
+      name:,
+      type: office_type
+    }
+  end
+
   private
 
   def opening_hours_as_json(opening_hours)
     return nil if opening_hours.nil?
 
     { opens: opening_hours.beginning.strftime("%H:%M:%S"), closes: opening_hours.ending.strftime("%H:%M:%S") }
   end
+
+  def relations_as_json
```

Comment:
💅 - would it be possible to rename this method, or `as_relation_json`?  Or maybe move `as_relation_json` underneath this method?  It took my (admittedly tired) brain a couple of mins to figure out what was being called where, so wondered if the ordering might help?